unit/testing-utils/load_java_class.cpp

```diff
@@ -18,6 +18,30 @@
 
 #include <java_bytecode/java_bytecode_language.h>
 
+
+/// Go through the process of loading, type-checking and finalising loading a
+/// specific class file to build the symbol table. The functions are converted
+/// using ci_lazy_methods (equivalent to passing --lazy-methods to JBMC)
+/// \param java_class_name: The name of the class file to load. It should not
+///   include the .class extension.
+/// \param class_path: The path to load the class from. Should be relative to
+///   the unit directory.
+/// \param main: The name of the main function or "" to use the default
+///   behaviour to find a main function.
+/// \return The symbol table that is generated by parsing this file.
+symbol_tablet load_java_class_lazy(
+  const std::string &java_class_name,
+  const std::string &class_path,
+  const std::string &main)
+{
+  auto language = new_java_bytecode_language();
+  free_form_cmdlinet lazy_command_line;
+  lazy_command_line.add_flag("lazy-methods");
+
+  return load_java_class(
+    java_class_name, class_path, main, std::move(language), lazy_command_line);
```

Comment:
Why not `new_java_bytecode_language()` directly here instead of using `std::move`?

Reply:
Good spot!